.coveragerc

```diff
@@ -1,5 +1,6 @@
+[run]
+source=evalml
```

Comment:
@ParthivNaresh what does this do?

Reply:
@dsherry This is [specifying](https://coverage.readthedocs.io/en/coverage-5.0/config.html#run) the source directory that the coverage is going to measure against